packages/react-components/react-rating/stories/src/Rating/RatingItemLabel.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Rating } from '@fluentui/react-components';
+
+export const ItemLabel = () => {
+  return <Rating itemLabel={num => `item #${num}`} />;
+};
+
+ItemLabel.parameters = {
+  docs: {
+    description: {
+      story: 'You can change the `aria-label` for each rating item using the `itemLabel` prop',
```

Comment:
I'm not sure if we need a story for this, but I'll leave that up to @microsoft/cxe-prg.

Reply:
Yeah, I agree, no need to make a separate story for this.

Reply:
removed the story as suggested